Makefile

```
@@ -53,22 +55,76 @@ setup:
+		exit 1; \
+	fi
+	@echo "Running bandit (sast-linter) for all modules..."
+	@if uv run bandit --configfile pyproject.toml --recursive . $(bandit_extra_args); then \
+		echo "bandit succeeded"; \
+	else \
+		echo "bandit failed"; \
+		exit 1; \
+	fi
+
+# Linting for staged files only (used by pre-commit hook)
+lint-staged:
```

checkmake (0.3.2)

[warning] Target body for "lint-staged" exceeds allowed length of 5 lines (11).

(maxbodylength)